src/app/header/header.component.html

```diff
@@ -159,7 +159,7 @@
         <div
           [ngClass]="
             numberOfActivePhase == 1
-              ? 'progress-bar progress-bar-striped progress-bar-animated bg-warning'
+              ? 'progress-bar bg-success'
```

Comment:
verstehe ich das richtig, dass du jetzt "phase 3" grün, statt gestreift anzeigst?
prinzipiell alles okay, zum thema codestyle & bootstrap:

1. die klasse progress-bar bleibt immer gleich, du kannst also string interpolation benutzen:
```[ngClass]="`progress-bar ${numberOfActivePhase === 1 ? 'bg-success' : 'bg-light'}`"```

2. du hast hier ein array von `phases`, du kannst einfach `phases.map((phase,idx) => {/* code goes here */})` machen und `numberOfActivePhase` mit `idx` vergleichen (wobei ich mir mit der genauen angular-notation nicht sicher bin, ist ewig her)

3. anstatt zwei separate layouts für desktop und mobile zu machen, kannst du die `.progress-bar`-divs alle in die gleiche `.row` packen und das bei den `.progress-bar`-divs mit den klassen `col-12 col-md-4` lösen. dann sparst du dir Zeilen 232 - 312

Reply:
ui ok - ja das sind gute Tipps, die ich mir mal anschaue 👍 Das mit dem -div classen hatte aus irgendeinem mir unerklärlichen Grund nicht funktioniert ... Aber ich merge jetzt ersteinmal und verbessere dann danach